torch_geometric/explain/utils/metrics.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from torch import Tensor
+
+
+def get_characterization_score(weights: Tensor,
```

Comment:
we can probably set a default weight here (1 : 1), since most of the time it's what people are going to use, unless there's prior knowledge.

Reply:
Makes sense, added :+1: